### PR TITLE
SLI-1146 Avoid relying on deprecated APIs

### PR DESCRIPTION
**Pull request by damien-urruty-sonarsource**

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -106,7 +106,7 @@ validate_task:
   eks_container:
     <<: *BUILDER_CONTAINER_DEFINITION
     cpu: 6
-    memory: 8G
+    memory: 10G
```
Comment (nquinquenel) [inline .cirrus.yml:109]: I wonder how this works, the instance used is t3.small, which has 2GB of memory (and 2 vCPU)

**Review by nquinquenel — APPROVED** [review]
LGTM

File: .cirrus.yml (hunk 1 above)
Comment (damien-urruty-sonarsource) [reply .cirrus.yml:109]: I will try to clarify that separately

Comment (damien-urruty-sonarsource) [reply .cirrus.yml:109]: From what I observe in the graphics on the build, this value is used as we are now a bit below 10G in effective usage

**Pull request merged**